lib/presentation/widget/upload_request_group/upload_request_group_viewmodel.dart

```diff
@@ -433,6 +436,35 @@ class UploadRequestGroupViewModel extends BaseViewModel {
     store.dispatch((UploadRequestGroupSetSearchResultAction([])));
   }
 
+  void openContextMenu(BuildContext context, UploadRequestGroup uploadRequestGroup, List<Widget> actionTiles, {Widget? footerAction}) {
+    store.dispatch(_handleContextMenuAction(context, uploadRequestGroup, actionTiles, footerAction: footerAction));
+  }
+
+  ThunkAction<AppState> _handleContextMenuAction(
+      BuildContext context,
+      UploadRequestGroup uploadRequestGroup,
+      List<Widget> actionTiles,
+      {Widget? footerAction}) {
+    return (Store<AppState> store) async {
+      ContextMenuBuilder(context)
+          .addHeader(ContextMenuHeaderBuilder(
+            Key('context_menu_header'),
+            UploadRequestGroupPresentationFile.fromUploadRequestGroup(uploadRequestGroup)).build())
+          .addTiles(actionTiles)
+          .addFooter(footerAction ?? SizedBox.shrink())
+          .build();
+      store.dispatch(UploadRequestGroupAction(Right(UploadRequestGroupContextMenuItemViewState(uploadRequestGroup))));
```

Comment:
Same question to above. What's usage of `UploadRequestGroupContextMenuItemViewState`?